Tools/autotest/sim_vehicle.py

```diff
@@ -528,8 +561,9 @@ def start_mavproxy(opts, stuff):
         if stuff["sitl-port"]:
             cmd.extend(["--sitl", simout_port])
 
-    # If running inside of a vagrant guest, then we probably want to forward our mavlink out to the containing host OS
-    ports = [p + 10 * cmd_opts.instance for p in [14550,14551]]
+    # If running inside of a vagrant guest, then we probably want to
+    # forward our mavlink out to the containing host OS
```

Comment:
Since you are modifying this, don't want to put another commit to move this down? It doesn't apply to the statement below it, but two lines below.

Reply:
Will do.